packages/grafana-ui/src/components/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory, useTheme } from '../../themes';
+
+export interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  header?: string | JSX.Element;
+  footer?: string | JSX.Element;
+}
+
+/**
+ * This is a simple InfoBox component, the api is not considered stable yet and will likely see breaking changes
+ * in future minor releases.
+ * @Alpha
+ */
+export const InfoBox = React.memo(
```

Comment:
@torkelo could you confirm this is right? (memo wrapper before forwardRef)

Reply:
looks good